junit-jupiter-engine/src/main/java/org/junit/jupiter/engine/discovery/TestMethodResolver.java

```diff
@@ -13,6 +13,7 @@
 import java.lang.reflect.Method;
 import java.util.function.Predicate;
 
+import org.junit.jupiter.api.Test;
```

Comment:
Unused import?

Reply:
It's used in the Javadoc and was broken previously.

Reply:
ok